extension-impl/fault-hystrix/src/main/java/com/alipay/sofa/rpc/hystrix/SofaHystrixCommand.java

```diff
@@ -47,6 +51,15 @@ public SofaHystrixCommand(FilterInvoker invoker, SofaRequest request) {
         this.request = request;
     }
 
+    @Override
+    public SofaResponse invoke() {
+        if (isCircuitBreakerOpen() && LOGGER.isWarnEnabled(invoker.getConfig().getAppName())) {
+            LOGGER.warnWithApp(invoker.getConfig().getAppName(), "Circuit Breaker is opened, method: {}#{}",
+                invoker.getConfig().getInterfaceId(), request.getMethodName());
```

Comment:
Does stopping all requests when `isCircuitBreakerOpen==true` ?  why just print warning log and continue?

Reply:
When the circuit breaker is opened, or the thread pool is rejected, etc., hystrix will execute the fallback methods which defined in https://github.com/alipay/sofa-rpc/blob/031a440462ea1b18891f7bfc7104df29fc77224e/extension-impl/fault-hystrix/src/main/java/com/alipay/sofa/rpc/hystrix/SofaHystrixCommand.java#L76-L78 and https://github.com/alipay/sofa-rpc/blob/031a440462ea1b18891f7bfc7104df29fc77224e/extension-impl/fault-hystrix/src/main/java/com/alipay/sofa/rpc/hystrix/SofaHystrixObservableCommand.java#L68-L101